roles-and-groups.md

```diff
@@ -0,0 +1,152 @@
+# Roles and Groups
+
+This file contains an overview of our rules for roles and groups.
+
+## Definition of general roles
+
+The Hiero project (and the LFDT) defines 3 different types of **roles**: **Contributors**, **Committers**, and **Maintainers**.
+Each person who contributes to Hiero must follow our [**Code of Conduct**](https://github.com/hiero-ledger/.github/blob/main/CODE_OF_CONDUCT.md).
+Those roles are not assigned globally for the Hiero project but per **project**.
+In most cases a project is a single GitHub repository, but it can also be a group of repositories that are related to each other.
+A person can have multiple roles in multiple projects.
+
+### Contributors
+
+Any person who contributes to a project is a Contributor. A contribution can be anything from opening an issue, to writing code, to writing documentation, etc.
+
+  - If a pull request has not received an approval within 2 days or a discussion has no activity for more than 2 days, the owner must reach out to the maintainers via our [hiero-general Discord channel](https://discord.lfdecentralizedtrust.org/).
+  - If a pull request or discussion has gone stale for more than 2 weeks, the repo maintainers may close them due to inactivity.
+
```

Comment:
Do we want to add any statements about issues here and the expectations?

Reply:
I would think issues and discussions would have a longer grace period is it? PRs would take priority or sure.
There is no mention in our LFDT guidelines on how long issues and discussions can stay idle but this is something our community can established. @rbarker-dev , @hendrikebbers , do you have any suggestions?

Reply:
I added some information on this in my latest patch